R/readStationary.R

```diff
@@ -1,37 +1,168 @@
 #' Load Stationary File as magclass object
 #'
-#' @return magclass object
+#' Reads in EDGE_TradMod.cs4r, which hast data for the SSP scenarios. In the convert function, the subset argument
+#' is used to specify the desired scenarios. Unspecified scenarios are copies of the SSP2 scenario.
 #'
-#' @author Antoine Levesque, Robin Hasse
+#' @return magclass object
+#' @order 1
 readStationary <- function() {
+  read.magpie("EDGE_TradMod.cs4r")
+}
 
-  scenarios <- list(SSPs  = paste0("SSP", 1:5),
-                    SSP2s = paste0("SSP2", c("_lowEn", "_highDemDEU")),
-                    SDPs  = paste0("SDP", c("", "_EI", "_MC", "_RC")))
-
-  data <- read.magpie("EDGE_TradMod.cs4r")
-  data[is.na(data)] <- 0
-  getSets(data) <- c("region", "year", "scenario", "item")
-
-  data <- mbind(
-    data,
-    toolAddDimensions(
-      x = mselect(data, scenario = "SSP2", collapseNames = TRUE),
-      dimVals = scenarios$SSP2s,
-      dimName = "scenario",
-      dimCode = 3.1
-    ),
-    toolAddDimensions(
-      x = mselect(data, scenario = "SSP1", collapseNames = TRUE),
-      dimVals = scenarios$SDPs,
-      dimName = "scenario",
-      dimCode = 3.1
-    )
-  )
+#' @rdname readStationary
+#' @order 2
+#' @param x MAgPIE object returned from readStationary
+#' @param subset A string, or vector of string designating the entries for the scenario dimension. If possible,
+#'  passed to [mrdrivers::calcGDP()]. Unspecified scenarios receive SSP2 data.
+convertStationary <- function(x, subset) {
```

Comment:
@johanneskoch94 moving read and convert into the same file is a big no for mrremind. Please don't do that.

Reply:
makes it hard to find functions for people who are browsing the repo online or are not familiar with commandline tools like `grep`